opentelemetry-sdk/src/opentelemetry/sdk/util/instrumentation.py

```diff
@@ -82,42 +84,56 @@ class InstrumentationScope:
     properties.
     """
 
-    __slots__ = ("_name", "_version", "_schema_url")
+    __slots__ = ("_name", "_version", "_schema_url", "_attributes")
 
     def __init__(
         self,
         name: str,
         version: Optional[str] = None,
         schema_url: Optional[str] = None,
+        attributes: Optional[dict] = None,
```

Comment:
No big deal, but noticed we are using `opentelemetry.util.types.Attributes` typing in several places for attributes typing. Maybe it makes sense to use it as well

Reply:
I think sdk and api should match

Reply:
fixed that.

Reply:
@vivek378521 

Have you made the change for this? It still says `Optional[dict]` instead of `Optional[Attributes]` for typing.

Reply:
Yes, did it now.